google-cloud-spanner/acceptance/spanner/client/transaction_test.rb

```diff
@@ -78,7 +80,7 @@
   end
 
   it "can rollback a transaction" do
```

Comment:
Thanks. Can you add a similar test for the normal commit case as well where multiple mutations are done and txn is committed instead of rollback as in this test.